File: C/4_vong_lap/bai_19.c
/*
Create on Tue Oct 07 2025 14:10
@author: ntdai

#20 VONG LAP for
	Cu phap:
		for(exp1; exp2; exp3){
			code thuc hien
		}
		Trong do:
			exp1: bieu thuc khoi tao
			exp2: bieu thuc dieu kien - dieu kien lap
			exp3: buoc nhay
*/

#include "stdio.h"

int main(){
	// xuat cac so tu 0->9 ra man hinh
	int i;
	for(i=0; i<=9; i++){
		printf("%d\n", i);
	}
	// xuat cac so tu 0->n ra man hinh
	int n;
	printf("nhap n = ");
	scanf("%d", &n);
	int k;
	for(k=0; k<=n; k++){
		printf("%d\n", k);
	}
}

File: python/part_2/bai_19.py
'''
Create Fri Oct 10 2025 21:10
@author: ntdai

#19 VÒNG LẶP LỒNG NHAU
'''
# Bài tập in bảng cửu chương
for j in range(2, 10):
    print("Bảng cửu chương", j)
    for i in range(1, 11):
        print("{0} * {1} = {2}".format(j, i, j*i))